include/tdnftypes.h

```diff
@@ -406,8 +405,9 @@ typedef struct _TDNF_REPOQUERY_ARGS
 
     /* query options */
     int nChangeLogs;
-    REPOQUERY_DEP_KEY depKey; /* list dependencies of this type, 0 => unset */
+    int *depKey; /* list dependencies of this type, 0 => unset */
```

Comment:
Instead of an array of integers, use a bit field.

Reply:
Changed to bitfield

Reply:
You declared it a pointer to `int`, but it's nowhere used as a pointer. For flags, use `unsigned int`.

Reply:
Also, use another name that reflects that it's a set of flags. `depKeys` (plural) is fine, or `depKeySet`.

Reply:
Done